test/e2e/e2e_test.go

```diff
@@ -429,14 +498,14 @@ func testPriority(t *testing.T, isPriorityClass bool) {
 	defer clientSet.SchedulingV1().PriorityClasses().Delete(ctx, lowPriorityClass.Name, metav1.DeleteOptions{})
 
 	// create two RCs with different priority classes in the same namespace
-	rcHighPriority := RcByNameContainer("test-rc-podlifetime-highpriority", testNamespace.Name, 5,
+	rcHighPriority := RcByNameContainer("test-rc-podlifetime-highpriority", testNamespace.Name, 3,
```

Comment:
I lowered that value because of the errors I was seeing about pending pods. I figured, since this line specifically would be run 4 times in a row, less pods created gives the tests following more room to schedule their pods. The alternative would be moving that 15s delay after a few tests to ensure the 5 pods from each replica set are fully deleted.